### PR TITLE
Bug 1413156 - Remove lodash forEach usage for arrays

### PR DESCRIPTION
**Pull request by CatEars**
I looked around for usages of _.forEach and found several places where it could be replaced trivially. Unfortunately I could not remove it entirely because it is used on both arrays and objects.

### CODE REVIEW

**Comment by edmorley** [comment]
Ah I've noticed this PR is from the master branch of your fork. This is fine for now, but for the future I'd recommend using a branch instead, otherwise you won't be able to have multiple PRs open at once, and you'll also have to reset your master branch should what lands deviate from what's in the PR now.

**Comment by CatEars** [comment]
Thanks for the tip @edmorley! I'll make sure to remember that.

**Comment by CatEars** [comment]
Yeah, I wasn't sure if I should target things that were objects with this PR. If I should then I'll just go ahead and use your suggestion, @jpumford.

File: ui/js/services/pinboard.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ treeherder.factory('thPinboard', [
         };
 
         var saveBugs = function (job) {
-            _.forEach(relatedBugs, function (bug) {
```
Comment (camd) [inline ui/js/services/pinboard.js]: ``relatedBugs`` here is actually an object, rather than an array.  So perhaps use ``Object.entries`` here.

**Comment by CatEars** [comment]
Applied your fix, @camd. Thanks for the review! I also managed to learn something new about ES6 destructors. :cake:

**Pull request merged**